src/layout/RegistrarList.tsx

```diff
@@ -45,14 +45,14 @@ function RegistrarList() {
       <View style={styles.registrarList}>
         <FlatList
           data={registrars}
-          renderItem={({item: registrar}) => {
+          renderItem={({item: registrar, index}) => {
             return (
               <View style={styles.registrarTeaserContainer}>
                 <RegistrarTeaser
                   key={registrar.account.toString()}
                   address={registrar.account.toString()}
                   fee={registrar.fee}
-                  index={registrar.index}
+                  index={index}
```

Comment:
This change will make it depend on the order of the array and not the registrar index itself.. If someone changes the sorting the index will be incorrect

Reply:
it was actually not a real index before also, it was the index of the map that was added to the element. https://github.com/litentry/litentry-app/pull/176/files#diff-94b39d441330c0bfb1c8d6c6713ec641959d0aa3425cfe624cce102fe66ef16bL12.
I will just add a todo to look into it later and merge this :)

Reply:
It is the real registrar index, the thing is that the api does not return the index as a field but in the correct order

Reply:
We should probably extract that into a function to make it descriptive or add some comments in the meantime

Reply:
I've changed it in 19d1b59. please check if you'd like that better.